doc/user_guide/user_guide.md

```diff
@@ -115,6 +115,14 @@ final Connection connectionB = factory.createConnection("PRIVILEGED_CONNECTION",
 
 As always the first parameter is the object name of the connection definition. Then there is the URL (e.g. a JDBC URL) and optionally username and password.
 
+### Creating Scripts
+
+Scripts are the main extension point for end-users. In Exasol you can for example define a Lua script like this:
+
+```sql
+final Script script = schema.createScript("HELLO_LUA", "print(\"Hello World\")");
+```
```

Comment:
Do you need to execute this script somehow after the object is created? When is it executed against the database?

I think it will be also convenient for users to see in this guide what kind of query is executed against the database when you create a script with this class.

Reply:
Added explanation section.